<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportNomorCSV</class>
 <widget class="QDialog" name="ImportNomorCSV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import CSV</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Import CSV</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0" colspan="4">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>    nomor,cluster(daerah/kota)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit_fileupload">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="pushButton_upload">
            <property name="text">
             <string>Jalankan Upload</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="pushButton_batal">
            <property name="text">
             <string>Tutup</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButton_pilih_csv">
            <property name="text">
             <string>Pilih File CSV</string>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Import CSV</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="4">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Valid Format CSV nomor sebagai berikut:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="4">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QCheckBox" name="checkBox_use_header">
            <property name="text">
             <string>Gunakan Header CSV (Kompatibilatas Dengan output MS EXCEL)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
